### ISSUE TITLE
Cmd + / conflict

### ISSUE THREAD

**Issue by projetmbc** [emmetio/emmet-atom#289]
Hello,
the keybinding Cmd + / conflict from Emmet is in conflict with the one of atom.

**Comment by projetmbc** [comment]
I have h-just seen that I can avoid this but I really think that by default the Emmet comment should be turned off.

**Comment by mech** [comment]
Hi also seeing this issue, the Emmet Cmd+/ is messing with all the other languages's comment :(

**Comment by sergeche** [comment]
![screen shot 2015-06-30 at 15 32 43](https://cloud.githubusercontent.com/assets/93595/8430842/474773f6-1f3d-11e5-8d7f-0dd18de3de7a.png)

**Comment by mech** [comment]
Hmms maybe I think it can be file-type aware, so that when open a JavaScript or Ruby file and do a comment, Emmet won't just hijack it and put a `<!-- -->` instead. It should be file-type sensitive, what do you think?

**Comment by projetmbc** [comment]
I totally agree but the real question is to know what is the advantage of Emmet comments regarding the Atom ones.

**Comment by sergeche** [comment]
http://docs.emmet.io/actions/toggle-comment/

**Comment by projetmbc** [comment]
Ok ! The video should be shown in the presentation of the package.

**Comment by JLLeitschuh** [comment]
1+ This is happening in my javascript files and it shouldn't be.

**Comment by jtzemp** [comment]
@sergeche Thanks for providing some context on how to turn it off and the usages. The thing is, Emmet is super helpful in HTML, but not in pure JS (or other non-SGML) files. I'd like to keep using emmet commenting in HTML files.

I think the issue comes up when you're in a file w/o any HTML and emmet is still taking over. Flipping the  setting back and forth depending on which file I'm in is less than ideal.

I'm in agreement with @mech. It'd be nice to have the commenting feature context aware. If I'm in an HTML context, I'd love emmet commenting. If not, I'd like to fall back on Atom's built-in commenting.

Thanks!

**Comment by drhayes** [comment]
:+1: Same here. I love emmet's commenting in straight HTML files, but in JS files seeing  comments is not helpful.

**Comment by sjurba** [comment]
To enable in HTML files only: https://github.com/emmetio/emmet-atom/issues/292#issuecomment-117940739
